packages/compass-e2e-tests/tests/collection-documents-tab.test.ts

```diff
@@ -487,8 +482,10 @@ FindIterable<Document> result = collection.find(filter);`);
     await browser.clickVisible(Selectors.SelectTableView);
 
     const document = await browser.$('.ag-center-cols-clipper .ag-row-first');
-    expect((await document.getText()).replace(/\s+/g, ' ')).to.match(
-      /^ObjectId\('[a-f0-9]{24}'\) 33 0$/
+    const text = (await document.getText()).replace(/\s+/g, ' ');
+    console.log({ text });
+    expect(text).to.match(
+      /^ObjectId\('[a-f0-9]{24}('\))? 33 0$/ // ') now gets cut off. sometimes.
```

Comment:
I don't know. The CSS that cuts the text short now behaves differently? Does that even make sense?